resources/install/debian/postinst

```diff
@@ -56,7 +56,7 @@ case "$1" in
             echo "JVB_SECRET=$JVB_SECRET" >> $CONFIG
             echo >> $CONFIG
             echo '# extra options to pass to the JVB daemon' >> $CONFIG
-            echo "JVB_OPTS=\"\"" >> $CONFIG
+            echo "JVB_OPTS=\"--apis=rest\"" >> $CONFIG
```

Comment:
Does this change the default value of the rest interface?

Reply:
Well, the intention is to turn off the component, as if we leave it, it fails health check. 
When I was initially adding the mucs support I added --no-component param for jigasi and jvb, but I think we merged only the jigasi part.

Reply:
My question is about the rest interface, not the component. Does a default debian installation come with rest enabled or disabled? It looks like it used to be disabled, and now it's enabled.

Reply:
Yeah, that's true ... I'm not sure how to disable the component another way. I will take a look.

Reply:
I think just passing `--apis=` should do it

Reply:
> I think just passing `--apis=` should do it

@bgrozev that doesn't work

```
JVB 2020-01-07 17:46:15.489 SEVERE: [33] org.jitsi.meet.ComponentMain.log() host-unknown, host:localhost, port:5347
org.xmpp.component.ComponentException: host-unknown
	at org.jivesoftware.whack.ExternalComponent.connect(ExternalComponent.java:243)
```

Reply:
Hum, but `--apis=,` works :) I will change it to that

Reply:
Nice find :)